src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -258,6 +257,11 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
       refreshApps()
     }
 
+    const loadBucketLocation = async (googleProject, bucketName) => {
+      const bucketLocation = await Ajax(signal).Workspaces.workspace(namespace, name).checkBucketLocation(googleProject, bucketName)
```

Comment:
Why is making more ajax calls preferred over adding global state?

Reply:
We discussed over slack that adding global state would cause more state subscriptions than we'd care for. The real fix here is to add bucketLocation to the workspace object in Rawls.